silkworm/rpc/http/connection.cpp

```diff
@@ -241,4 +244,14 @@ void Connection::set_cors(boost::beast::http::response<Body>& res) {
     res.set("Access-Control-Max-Age", "600");
 }
 
+std::string Connection::get_date_time() {
+    const boost::posix_time::ptime now = boost::posix_time::second_clock::local_time();
+    const auto date = now.date();
+    const auto time = now.time_of_day();
+    std::stringstream ss;
+    ss << date.day_of_week() << ", " << date.day() << " " << date.month() << " " << date.year() << " "
+       << time.hours() << ":" << time.minutes() << ":" << time.seconds() << " GMT";
+    return ss.str();
+}
```

Comment:
I suggest using `absl::Now` and `absl::FormatTime` to handle date/time formatting, as done in [log](https://github.com/erigontech/silkworm/blob/master/silkworm/infra/common/log.cpp#L137)

Reply:
ok